oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -151,6 +155,44 @@ public AccessToken refreshAccessToken() throws IOException {
     return new AccessToken(accessToken, new Date(expiresAtMilliseconds));
   }
 
+  /**
+   * Returns an Google Id Token from the metadata server on ComputeEngine.
+   * 
+   * @param targetAudience The aud: field the IdToken should include.
+   * @param options        List of Credential specific options for for the
```

Comment:
We don't indent like this since it's too hard for autoformatters to maintain

Reply:
ok, removed indent (vs code's java 'autoformatter' does this type of indents for you)